docs/api/web-contents.md

```diff
@@ -1035,15 +1035,15 @@ Detaches the debugger from the `webContents`.
 
 Send given command to the debugging target.
 
-#### Event: 'detach'
+### Event: 'detach'
```

Comment:
This event is for the debugger which is already `####` so going back to `###` might make it seem like they refer to `webContents`, instead of `webContents.debugger`.